Gemfile

```diff
@@ -2,32 +2,22 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
+
 branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
 gem 'solidus', github: 'solidusio/solidus', branch: branch
 
-if branch == 'master' || branch >= 'v2.3'
```

Comment:
Is it no longer needed include the `rails` version?

Reply:
Nope, it will install what solidus has as dependency (newest possible)